src/openforms/js/components/form/map.js

```diff
@@ -62,6 +62,80 @@ const EDIT_FORM_TABS = {
         HIDDEN,
         CLEAR_ON_HIDE,
         {...IS_SENSITIVE_DATA, defaultValue: true},
+        {
+          type: 'number',
+          key: 'defaultZoom',
+          label: 'Default zoom level.',
+          description: 'Default zoom level for the leaflet map.',
+          validate: {
+            min: 1,
+            max: 18,
+          },
+        },
+        {
+          type: 'columns',
+          input: false,
+          label: 'starting location',
+          key: 'startPop',
+          tooltip: 'The Latitude and Longitude of the starting point of the map component.',
+          defaultValue: [{lat: '', lng: ''}],
+          columns: [
+            {
+              components: [
+                {
+                  label: 'Latitude',
+                  key: 'lat',
```

Comment:
this should be properly namespaced - I'd go with `initialCenter.lat` here instead of `lat`, which results in the JSON definition like this:

```json
{
  "type": "map",
  "initialCenter": {
    "lat": 52.1326332,
    "long": 5.291266,
  }
}
```